runtime/src/topo/hwloc/topo-hwloc.c

```diff
@@ -298,15 +310,31 @@ void getNumCPUs(void) {
   numCPUsLogAcc = hwloc_bitmap_weight(logAccSet);
   CHK_ERR(numCPUsLogAcc > 0);
 
-  hwloc_bitmap_free(logAccSet);
-
   //
   // all PUs
   //
   numCPUsLogAll = hwloc_get_nbobjs_by_type(topology, HWLOC_OBJ_PU);
   CHK_ERR(numCPUsLogAll > 0);
 }
 
+// Fills the cpus array with the physical indices of the CPUs in the cpuset.
```

Comment:
`physical indices` probably isn't quite right since this is using hwloc numbering and not the physical CPU ids.

Reply:
I was just going off the `hwloc` documentation for `hwloc_cpuset_t`
```
A CPU set is a bitmap whose bits are set according to CPU physical OS indexes.
```
I got bit by this so I wanted to document it.

Reply:
Let's use the full hwloc terminology then and go with something like "Fills the cpus array with the hwloc cpuset (a bitmap whose bits are set according to CPU physical OS indexes)"

Reply:
I'll fix it.